front/app/containers/Admin/projects/project/traffic/index.tsx

```diff
@@ -4,20 +4,26 @@ import { useParams } from 'react-router-dom';
 
 import usePhases from 'api/phases/usePhases';
 
+import { START_DATE_PAGEVIEW_AND_EXPANDED_SESSION_DATA_COLLECTION as LOWER_BOUND } from 'containers/Admin/dashboard/constants';
+
 import TrafficDatesRange from './TrafficDatesRange';
 
 const ProjectTraffic = () => {
   const { projectId } = useParams() as { projectId: string };
   const { data: phases } = usePhases(projectId);
+  if (!phases) return null;
 
-  const startOfFirstPhase = phases?.data[0]?.attributes.start_at;
-  const endOfLastPhase =
-    phases?.data[phases.data.length - 1]?.attributes.end_at;
+  const startOfFirstPhase = phases.data[0]?.attributes.start_at;
+  const endOfLastPhase = phases.data[phases.data.length - 1]?.attributes.end_at;
+
+  const startDate =
```

Comment:
For the project-specific visitor data / "traffic" data we use the the date we started collecting pageview / more rich session data, back in November 2024, EXCEPT if the first phase started after that

Reply:
This is also the minimum selectable date